app/config/config.exs

```diff
@@ -23,7 +25,7 @@ config :logger, :console,
   metadata: [:request_id]
 
 config :sparkpost,
-  api_key: "TODO SUPPLY KEY",
+  api_key: System.get_env("SPARKPOST_API_KEY") || "${SPARKPOST_API_KEY}",
```

Comment:
I don't think the`"${....}"` will do any magic.  If you want to be able to configure them at runtime, ping me and I'll show you how.

Reply:
Ugh, I forgot to remove that.